README.md

```diff
@@ -1,21 +1,192 @@
+
+1. [APIs](#apis)
+1. [Tools](#tools)
+1. [Execution Environment](#execution-environment)
+1. [Issue Reporting](#issue-reporting)
+1. [Contributing](#contributing)
+1. [Author](#author)
+1. [License](#license)
+
+## APIs
+
+## Expenses API - Secured
+
+The Expenses API is a simple api that is secured using the [express-oauth2-bearer](https://github.com/auth0/express-oauth2-bearer) module.
+
+### Environment Variables
+
+| Variable          | Value                                | Default |
+| ----------------- | ------------------------------------ | ------- |
+| ISSUER_BASE_URL   | https://your-tenant.region.auth0.com | ❌      |
+| ALLOWED_AUDIENCES | **https://expenses-api**             | ✅      |
+| PORT              | **5000**                             | ✅      |
+
+#### Run Local:
+
+```bash
+ISSUER_BASE_URL=https://your-tenant.region.auth0.com \
+npm run expenses-api:secure:start
+```
+
+#### Vercel Deploy:
+
+```
+https://vercel.com/new/git/external
+  ?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fmaster%2Fapis%2Fexpenses-api-secured
+  &env=
+    ISSUER_BASE_URL,
+    VERCEL_URL,
+    ALLOWED_AUDIENCES=https://expenses-api
+  &project-name=expenses-api
+  &repository-name=expenses-api
+```
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fmaster%2Fapis%2Fexpenses-api-secured&env=ISSUER_BASE_URL,ALLOWED_AUDIENCES=https://expenses-api,VERCEL_URL&project-name=expenses-api&repository-name=expenses-api)
+
+## Expenses API - Unsecured
+
+The Unsecured Expenses API is a simple api that is intended to be used as a starting point for hands on labs.
+
+### Environment Variables
+
+| Variable          | Value                                | Default |
+| ----------------- | ------------------------------------ | ------- |
+| ISSUER_BASE_URL   | https://your-tenant.region.auth0.com | ❌      |
+| ALLOWED_AUDIENCES | **https://expenses-api**             | ✅      |
+| PORT              | **5000**                             | ✅      |
+
+#### Run Local:
+
+```bash
+ISSUER_BASE_URL=https://your-tenant.region.auth0.com \
+npm run expenses-api:start
+```
+
+#### Vercel Deploy:
+
+```
+https://vercel.com/new/git/external
+  ?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fmaster%2Fapis%2Fexpenses-api-unsecured
+  &env=
+    ISSUER_BASE_URL,
+    VERCEL_URL,
+    ALLOWED_AUDIENCES=https://expenses-api
+  &project-name=expenses-api-unsecured
+  &repository-name=expenses-api-unsecured
+```
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https%3A%2F%2Fgithub.com%2Fauth0%2Fauth0-product-education-labs%2Ftree%2Fmaster%2Fapis%2Fexpenses-api-unsecured&env=ISSUER_BASE_URL,ALLOWED_AUDIENCES=https://expenses-api,VERCEL_URL&project-name=expenses-api-unsecured&repository-name=expenses-api-unsecured)
```

Comment:
Entered issuer base URL and audience but Failed with error `Unexpected Error (POST https://vercel.com/api/v1/integrations/push-to-repo)`

Reply:
I think this might be because the files are currently in a PR.

Reply:
I don't really see a way around this. If I change the link to work in a PR it will not work in the master branch. We will have to test it once merged.